src/cadenza.js

```diff
@@ -164,13 +166,20 @@ export class CadenzaClient {
    * @param {boolean} [options.hideWorkbookToolBar] - Whether to hide the workbook toolbar.
    * @param {string} [options.mediaType] - Set to "application/pdf" for Jasper Report views
    *     to show the PDF directly, without any Cadenza headers or footers.
+   * @param {GlobalId} [options.highlightGlobalId] - The id of the item to expand in the navigator tree.
```

Comment:
If the item is not a folder, it gets highlighted instead of expanded, right? The parameter description should reflect that, e.g. "The id of the item to expand or highlight in the navigator tree."